crates/bevy_render/src/render_resource/texture.rs

```diff
@@ -22,7 +36,7 @@ impl Texture {
         self.id
     }
 
-    /// Creates a view of this texture.
+    /// Creates a [`TextureView`](crate::render_resource::TextureView) of this texture.
```

Comment:
```suggestion
    /// Creates a [`TextureView`] of this texture.
```
This shouldn't be needed here or elsewhere; the type's already in scope.